corehq/apps/enterprise/api/resources.py

```diff
@@ -388,3 +388,33 @@ def dehydrate(self, bundle):
 
     def get_primary_keys(self):
         return ('form_id', 'submitted',)
+
+
+class CaseManagementResource(ODataEnterpriseReportResource):
+    domain = fields.CharField()
+    num_applications = fields.IntegerField()
+    num_surveys_only = fields.IntegerField()
+    num_cases_only = fields.IntegerField()
+    num_mixed = fields.IntegerField()
+
+    REPORT_SLUG = EnterpriseReport.CASE_MANAGEMENT
+
+    def get_report_task(self, request):
```

Comment:
this function can be removed as I moved `get_report_task` to base class in https://github.com/dimagi/commcare-hq/pull/35468/commits/5609175b0896d15809039947dbf8bcd1da9d2cd3